### PR TITLE
Test case to verify PVC clone from GUI

### PR DESCRIPTION
**Pull request by jilju**
Test case to verify PVC clone from GUI

### CODE REVIEW

File: tests/ui/test_pvc_ui.py [hunk 1]
```diff
@@ -15,16 +15,16 @@
 logger = logging.getLogger(__name__)
 
 
+@ui
+@skipif_ocs_version("<4.6")
+@skipif_ui_not_support("pvc")
```
Comment (am-agrawa) [inline tests/ui/test_pvc_ui.py:20]: @OdedViner Do we still use this marker?

File: tests/ui/test_pvc_ui.py [hunk 2]
```diff
@@ -196,3 +196,86 @@ def test_create_resize_delete_pvc(
...
+        logger.info("Verifying cloned PVC details in UI")
+        pvc_ui_obj.verify_pvc_ui(
+            pvc_size=pvc_size,
+            access_mode=clone_access_mode,
+            vol_mode=vol_mode,
+            sc_name=sc_name,
+            pvc_name=clone_pvc_name,
+            project_name=project_name,
+        )
+        logger.info("Verified cloned PVC details in UI")
```
Comment (am-agrawa) [inline tests/ui/test_pvc_ui.py:281]: After cloned PVC verification, aren't we suppose to delete the PVC/Cloned PVC and the test namespace?

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 3]
```diff
@@ -239,3 +239,52 @@ def delete_pvc_ui(self, pvc_name, project_name):
         logger.info("Confirm PVC Deletion")
         self.do_click(self.pvc_loc["confirm_pvc_deletion"])
         time.sleep(1)
+
+    def pvc_clone_ui(
+        self,
+        project_name,
+        pvc_name,
+        clone_access_mode=constants.ACCESS_MODE_RWO,
+        clone_name=None,
```
[the remark this answers is not in the document]
Comment (jilju) [reply ocs_ci/ocs/ui/pvc_ui.py]: Done

File: ocs_ci/ocs/ui/pvc_ui.py [hunk 4]
```diff
@@ -239,3 +239,52 @@ def delete_pvc_ui(self, pvc_name, project_name):
         logger.info("Confirm PVC Deletion")
         self.do_click(self.pvc_loc["confirm_pvc_deletion"])
         time.sleep(1)
+
+    def pvc_clone_ui(
+        self,
+        project_name,
+        pvc_name,
+        clone_access_mode=constants.ACCESS_MODE_RWO,
```
[the remark this answers is not in the document]
Comment (jilju) [reply ocs_ci/ocs/ui/pvc_ui.py]: Done

File: tests/ui/test_pvc_ui.py (hunk 2 above)
Comment (jilju) [reply tests/ui/test_pvc_ui.py:281]: @am-agrawa Yes, teardown_factory is used in line 244 and 267. Project_factory finalizer will delete the namespace.

Comment (am-agrawa) [reply tests/ui/test_pvc_ui.py:281]: And what about PVC deletion? Aren't we doing that?

Comment (jilju) [reply tests/ui/test_pvc_ui.py:281]: Yes, teardown_factory will do that

**Review by am-agrawa — APPROVED** [review]
LGTM

**Review by OdedViner — DISMISSED** [review]

**Review by OdedViner — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**